google/auth/credentials.py

```diff
@@ -24,6 +24,8 @@
 from google.auth import metrics
 from google.auth._refresh_worker import RefreshThreadManager
 
+DEFAULT_UNIVERSE_DOMAIN = "googleapis.com"
```

Comment:
I think there are other instances of this, e.g. in external account. Can you consolidate? 

Also, "googleapis.com" is used below. Should be replaced there & elsewhere

Reply:
Great. In the beginning I wanted to do those refactoring together and I realized the PR is big.
Then i planned to open a PR only to refactor the `DEFAULT_UNIVERSE_DOMAIN` at once.